blocks/article-hero/article-hero.css

```diff
@@ -12,7 +19,24 @@
     gap: 30px;
 }
 
-.article-hero .hero-overlay  {
+div.hero-overlay-smaller-screen > a::before,
+.article-hero div.hero-overlay > a::before{
+  content: "By: ";
+}
+
+.article-hero div.hero-overlay > a{
+  color: #a0a0a0
+}
+
+.article-hero div.hero-collections > a {
+  color: var(--accent-color);
+}
+
+div.hero-overlay > a:hover {
+  color: var(--link-color-default-content);
+}
```

Comment:
This is too broad, I think this should only apply to the author. I think we can just remove this and use the default link color for now.